sdk/storage/Azure.Storage.Blobs.Batch/src/Azure.Storage.Blobs.Batch.csproj

```diff
@@ -14,8 +14,6 @@
       This client library allows you to batch multiple Azure Blob Storage operations in a single request.
       For this release see notes - https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs.Batch/README.md and https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs.Batch/CHANGELOG.md
       in addition to the breaking changes https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs.Batch/BreakingChanges.txt
-      For this release see notes - https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs.Batching/README.md and https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs.Batching/Changelog.txt
```

Comment:
Worth yanking the preview3 link above while we're doing this?

Reply:
What would be the replacement?

Reply:
I'd just kill the whole section unless we have a new aka.ms link?

Reply:
Done.